src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -1037,29 +1035,26 @@ private static bool IsCollectionHelper(Type type, [NotNullWhen(true)] out Type?
                 itemType = type.GetElementType()!;
                 return true;
             }
-            DataContract? dataContract;
-            return IsCollectionOrTryCreate(type, false /*tryCreate*/, out dataContract, out itemType, constructorRequired);
+            return IsCollectionOrTryCreate(type, false /*tryCreate*/, out _, out itemType, constructorRequired);
```

Comment:
As long as you're changing these, we could use named arguments rather than putting names in comments.

Reply:
ok